config/helm/aws-node-termination-handler/README.md

```diff
@@ -68,6 +68,7 @@ The configuration in this table applies to all AWS Node Termination Handler mode
 | `affinity`                         | Affinity settings for pod assignment. In IMDS mode this has a higher priority than `daemonsetAffinity` (for backwards compatibility) but shouldn't be used.                                                                                                                                                                                                                            | `{}`                                                  |
 | `tolerations`                      | Tolerations for pod assignment. In IMDS mode this has a higher priority than `daemonsetTolerations` (for backwards compatibility) but shouldn't be used.                                                                                                                                                                                                                               | `[]`                                                  |
 | `extraEnv`                         | Additional environment variables for the _aws-node-termination-handler_ container.                                                                                                                                                                                                                                                                                                     | `[]`                                                  |
+| `envFrom`                          | Additional environment variables for the _aws-node-termination-handler_ container.                                                                                                                                                                                                                                                                                                     | `[]`                                                  
```

Comment:
Thanks for adding this. Could you update the description for one or both of the `extraEnv` and `envFrom` parameters to make clear the distinction between them? Some things to consider may be where the parameters are used in the chart and whether their syntax is different.